src/methods.jl

```diff
@@ -39,6 +39,16 @@ function subtract_bias!(frame::AbstractArray, bias_frame::AbstractArray)
 end
 
 
+"""
+    subtract_bias!(frame::AbstractArray, bias_frame::ImageHDU)
```

Comment:
I don't know what's going wrong here, it looks perfectly fine in my editor. (same on line 76) Is it a GH glitch?

Reply:
I don't see what you're talking about.

Reply:
Fixed it! It was an indentation error from my end.